### PR TITLE
Add description of data fetching/munging

### PR DESCRIPTION
**Pull request by sleepycat**
Adding a description of where we got the data and what we did to it...
also the full command, in all it's bash-y glory.

### CODE REVIEW

File: index.Rmd
Comment (caromimo) [inline index.Rmd]: What happened here? Looks like a typo.

File: index.Rmd
Comment (caromimo) [inline index.Rmd]: It is good practice to report the date on which data was extracted. Do you mind reviewing to include it in your description?

File: index.Rmd
Comment (caromimo) [inline index.Rmd]: I would include this sentence before the actual line of code - it provides context to understand the code - do we also want to specify that this was done at the command line or is this obvious already?

File: index.Rmd
Comment (caromimo) [inline index.Rmd]: I had the previous wording because the column are not named at this stage (this was done in the cleaning of the CSV detailed in the next section). How about: "The CSV contained the name of the package, the version and a time stamp."?

File: index.Rmd
Comment (caromimo) [inline index.Rmd:170]: trailing spaces and trailing lines... an abomination.... :)

**Review by caromimo — CHANGES_REQUESTED** [review]
Thank you for your contribution. Please see comments and suggestions. Once addressed, I will happily merge!

File: index.Rmd
Comment (sleepycat) [reply index.Rmd]: Good catch. Fixed!

File: index.Rmd
Comment (sleepycat) [reply index.Rmd]: If the date is to act as a version the file, the SHA in the GitHub perma-link actually refers to the manifest file as it exists in that specific commit: 
`https://github.com/canada-ca/tracker/blob/003a3d6e0452fb9a248a7d51fd5556bc03dfbcc6/frontend/package.json`
I think that achieves the goal, just in a different form.

If we are thinking about the version of the stuff returned from the api... that's pretty stable and would need a `/v2/` in the url or something if it were to change.

I'm happy to add the date if that's a stylistic thing/convention you want to follow, but if the concern is reproducability I think that aspect is handled. 
Thoughts?

File: index.Rmd
Comment (sleepycat) [reply index.Rmd:170]: :grimacing:

File: index.Rmd
Comment (caromimo) [reply index.Rmd]: Thanks

File: index.Rmd
Comment (caromimo) [reply index.Rmd]: It is for reproducibility - if someone wants to reproduce this analysis they should be able to do so by following the methodology - if we need to add more steps then let's do so - if those include a commit, then I agree that it would serve as a version control so the person could download the same version as the one used in the analysis. I leave it to you to expand on this as you see fit. Thanks.

File: index.Rmd
Comment (caromimo) [reply index.Rmd]: Thanks for addressing my comment. I think the sentence got duplicated for some reason.

File: index.Rmd
Comment (sleepycat) [reply index.Rmd]: OK, so I added another commit that downloads the data reproducibly and replaces the existing `data/raw/nodejs.csv` with one that is exactly what was produced from the package.json and placed it in `data/processed/nodejs.csv`. The text has been updated to reflect that.

File: index.Rmd
Comment (sleepycat) [reply index.Rmd]: :brain: Probably best to optimize for clarity.
I moved the sentence and specified that this is a bash script and added a link to clarify.

File: index.Rmd
Comment (sleepycat) [reply index.Rmd]: Fixed!

File: index.Rmd
Comment (caromimo) [reply index.Rmd]: LGTM

File: index.Rmd
Comment (caromimo) [reply index.Rmd]: Thanks

File: index.Rmd
Comment (caromimo) [reply index.Rmd]: Thanks

**Comment by caromimo** [comment]
Thanks for your contribution.

**Comment by caromimo** [comment]
Merging the pull request

**Pull request merged**
